docs/modules/ROOT/pages/vshn-managed/postgresql/usage.adoc

```diff
@@ -5,32 +5,176 @@ The latter references the `postgres-creds` secret that will contain the access k
 
 [source,yaml]
 ----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: app1-prod
+  labels:
+    name: app1-prod
+---
 apiVersion: vshn.appcat.vshn.io/v1
 kind: VSHNPostgreSQL
 metadata:
-  name: my-postgres-example
-  namespace: my-namespace
+  name: postgresql-prod
+  namespace: app1-prod
 spec:
   parameters:
     service:
       majorVersion: "15"
+    backup:
+      schedule: '0 22 * * *'
   writeConnectionSecretToRef:
     name: postgres-creds-connection
 ---
 apiVersion: v1
 kind: Pod
 metadata:
   name: postgres-client
+  namespace: app1-prod
 spec:
   containers:
   - name: postgres
     image: postgres:15
+    resources:
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
     command:
       - /bin/sh
       - -c
     args:
-      - PGPASSWORD=$POSTGRESQL_PASSWORD psql -h $POSTGRESQL_HOST -d $POSTGRESQL_DB -U $POSTGRESQL_USER -l
+      - PGPASSWORD=$POSTGRESQL_PASSWORD psql "sslmode=verify-ca sslrootcert=/etc/secret-volume/ca.crt host=$POSTGRESQL_HOST port=$POSTGRESQL_PORT dbname=$POSTGRESQL_DB" -U $POSTGRESQL_USER -l
     envFrom:
       - secretRef:
           name: postgres-creds-connection
-  restartPolicy: Never
+    volumeMounts:
+    - name: secret-volume
+      readOnly: true
+      mountPath: "/etc/secret-volume"
+  volumes:
+  - name: secret-volume
+    secret:
+      defaultMode: 0600
+      secretName: postgres-creds-connection
+  restartPolicy: OnFailure
+----
+
+== Debug instance by yourself
+
+You can easily check what happened during creation by checking the status fields. PostgreSQL by VSHN is composed from many resources, responsible for
+cluster creation, bucket creation, backups and uploads to cloud, each of those components might break or be misconfigured (f.e typo). You can check what happend by running
+
+```
+# example assumes reuse of names and namespace from example above
+kubectl -n app1-prod describe vshnpostgresqls.vshn.appcat.vshn.io postgresql-prod 
+```
+
+Example output looks as follows:
+
+```
+Status:
+  Certificate Debug:
```

Comment:
Why is this called "... Debug"? IMHO that's a very strange naming for a status field, I've never seen that in any Kubernetes resource this way. "Debug" implies to me that I increased the log level somewhere and now it tells my messages from a debugging log level, which I would not want to see in production.

Reply:
I called that `...Debug` because purpose of that field is to help our customers debug their instances

Reply:
I must admit: I don't like it at all. Up to a point where I'd like to ask to change that. My proposal would be to call that "... Conditions". Because as it looks like it's a copy of the "Conditions" field from other resources, is it?

Reply:
No problem, I'll create a fastlane ticket for that